### PR TITLE
De-red-ify more crosses

### PR DESCRIPTION
**Pull request by MACMAN2003**
## About the PR

un-redded even more red crosses
## Why / Balance

i missed a few
## Media

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

### CODE REVIEW

**Review by QuietlyWhisper — APPROVED** [review]
thanks

File: Resources/Textures/_RMC14/Interface/health_hud.rsi/hudhealthy.png
Comment (QuietlyWhisper) [inline Resources/Textures/_RMC14/Interface/health_hud.rsi/hudhealthy.png:1]: I just remembered you'll need to fix this image because it got scaled down

**Review by QuietlyWhisper — CHANGES_REQUESTED** [review]

**Comment by MACMAN2003** [comment]
i think it's ready??? (git is evil i hate it)

**Review by QuietlyWhisper — APPROVED** [review]
Thanks!

**Pull request merged**